fly/commands/clear_resource_cache.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/fly/commands/internal/flaghelpers"
+	"github.com/concourse/concourse/fly/rc"
+)
+
+type ClearResourceCacheCommand struct {
+	Resource 	 flaghelpers.ResourceFlag `short:"r" long:"resource" required:"true" value-name:"PIPELINE/RESOURCE" description:"Name of a resource to clear cache"`
+	Version      *atc.Version             `short:"v" long:"version"                  value-name:"VERSION"           description:"Version of the resource to check from, e.g. digest:sha256@..."`
+}
+
+func (command *ClearResourceCacheCommand) Execute(args []string) error {
+
+	target, err := rc.LoadTarget(Fly.Target, Fly.Verbose)
+	if err != nil {
+		return err
+	}
+
+	err = target.Validate()
+	if err != nil {
+		return err
+	}
+
```

Comment:
I'm wondering if we want to display some sort of warning message before we go ahead and delete the resource cache? Maybe something like the one for `clear-task-cache` https://github.com/concourse/concourse/blob/2000fa3b136cdd95fc0b8ad29bcfc7efb36273dd/fly/commands/clear_task_cache.go#L29-L35

What do you think? 🤔

Reply:
I think is a good idea, is better to keep safe with this kind of things